### PR TITLE
Update pooling.py added max_pool3d function

### LINKED ISSUE #23251: max_pool3d

### PR DESCRIPTION
**Pull request by themohal**
# PR Description

## Related Issue

Close #23251 

## Checklist

- [x] Did you add a function?
- [ ] Did you add the tests?
- [ ] Did you follow the steps we provided?

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by themohal** [comment]
just lint check failing

File: ivy/functional/frontends/paddle/nn/functional/pooling.py [hunk 1]
```diff
@@ -99,6 +99,86 @@ def avg_pool2d(
...
+    return ivy.pool(
+        input,
+        mode="max",
+        data_format=data_format,
+        window_shape=(kernel_size, kernel_size, kernel_size),
+        stride=stride,
+        padding=padding,
+        dilation=dilation,
+        method="floor",
+    )
```
Comment (dhanush-2501) [inline ivy/functional/frontends/paddle/nn/functional/pooling.py:181]: This redundancy can be optimised with an single line if statement.

File: ivy/functional/frontends/paddle/nn/functional/pooling.py [hunk 2]
```diff
@@ -99,6 +99,86 @@ def avg_pool2d(
...
+    stride: The stride of the pooling operation. This can be a single number or a tuple of three numbers, one for each dimension of the input tensor. If None, defaults to kernel_size.
+    padding: The padding of the pooling operation. This can be a single number or a tuple of three numbers, one for each dimension of the input tensor.
+    dilation: The dilation of the pooling operation. This can be a single number or a tuple of three numbers, one for each dimension of the input tensor.
+    ceil_mode: Whether to use ceil or floor mode for calculating the output shape.
+    data_format: The data format of the input tensor. Can be "NCHW" or "NHWC".
+    name: The name of the operation.
+
+  Returns:
+    A tensor of the same shape as the input tensor, except that the depth, height, and width dimensions are all reduced by the stride size.
+  """
```
Comment (dhanush-2501) [inline ivy/functional/frontends/paddle/nn/functional/pooling.py:130]: We don't write doc strings in the frontend api

**Pull request closed without merging**